extensions/sockets/src/main/java/org/mule/module/socket/internal/ConnectionEvent.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.socket.internal;
+
+import org.mule.module.socket.api.source.SocketAttributes;
+
+public class ConnectionEvent
```

Comment:
I though about that but I need information from the sockets in order to build de Attributes that are going to travel with the payload in the Message. I can build the MuleMessage directly with all the socket information but that would require some coupling the client knowing about the MuleMessage. Maybe is not as bad as having this phony Message that only helps to build the MuleMessage